### DISCUSSION TITLE
How do I select a specific Julia version when using juliacall?

### DISCUSSION THREAD

**Discussion by LilithHafner** [cjdoris/PythonCall.jl#506] [Q&A] [answered]
I want to run Julia 1.8 via juliacall to reproduce a bug (https://github.com/SciML/diffeqpy/issues/135), is there an environment variable or something I can set for that?

**Comment by cjdoris** [comment, marked as answer]
See here on how to declare compat bounds on Julia in JuliaPkg: https://github.com/JuliaPy/pyjuliapkg?tab=readme-ov-file#declare-dependencies

Or if you just want to use a version of Julia you already have installed, set the env var [`PYTHON_JULIAPKG_EXE`](https://github.com/JuliaPy/pyjuliapkg?tab=readme-ov-file#configuration)

**Reply by LilithHafner** [reply]
Thanks!
